runtime-modules/content/src/lib.rs

```diff
@@ -2304,6 +2308,23 @@ decl_module! {
             Self::deposit_event(RawEvent::NftBought(video_id, participant_id));
         }
 
+        /// Only Lead can toggle nft issuance limits constraints
```

Comment:
```suggestion
        /// Only Council can toggle nft issuance limits constraints
```

Reply:
I added the suggested comment but only `root` can toggle the variable at the moment